.github/workflows/CI.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           node-version: 14
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --immutable-cache # lock changes should already be committed.
```

Comment:
docs say you are supposed to use --immutable as a replacement for --frozen-lockfile, not --immutable-cache